README.md

```diff
@@ -35,3 +35,9 @@ That work was presented [here](https://www.youtube.com/watch?v=RS1aiQqgUTA).
 Questions? Comments? Ideas?
 Come discuss `ko` with us in the `#ko-build` channel on the [Kubernetes Slack](https://slack.k8s.io)!
 See you there!
+
+### Community Meetings
+
+We have a bi-weekly community meeting on [Thursdays @ 6:00pm UTC time](https://dateful.com/eventlink/2793571750). The main goal of these meetings is that we want to hear from you! We want to know what you're using `ko` for, what you'd like to see in `ko`, how we can make `ko` better for you and with any remaining time we can go through open issues/PRs.
```

Comment:
Is this something we should put on the website, instead of the README?

Maybe a separate ko.build/community page?

Reply:
The ko repository is the first page people will visit before the ko.build website because we have always shared the ko repository everywhere. So, it'd be better to do the same ko.build website to increase the count of visitors to the website. So I recommend adding the meeting details to both website and README.

Reply:
Instead of duplicating content, can we link from the README to ko.build/community?